lib/acx/enforcer_server.ex

```diff
@@ -347,6 +356,17 @@ defmodule Acx.EnforcerServer do
     end
   end
 
+  def handle_call({:reset_configuration, {cfile, adapter}}, _from, enforcer) do
```

Comment:
What if we also had a condition where adapter has already been set with `set_persist_adapter/2`? I noticed `Enforcer.set_persist_adapter/2` adds `persist_adapter` to the state and could be pattern matched on like this `def handle_call({:reset_configuration, cfile}, _from, %{persist_adapter: adapter} enforcer) do` then we wouldn't have to pass around the adapter once it is already set.